### PR TITLE
Improve README.md

### PR DESCRIPTION
**Pull request by gwenneg**
Here are a few suggestions about some `README.md` improvements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,22 +18,22 @@
...
-Also, the OpenAPI ecosystem is very large (https://openapi.tools/) with a lot of tools with different quality level. Consequently, to set up a complete development environment, many tools has to be installed and upgraded regularly.
+Although using OpenAPI comes with a lot of advantages (documentation and code generation, faster API publishing, teams collaboration promotion...), industrializing it remains complicated due to certain shortcomings. Using a single OpenAPI file for several contexts (public and private usage for example) is not supported by the current release of OpenAPI. Using the external references to design your API from several files to facilitate the maintainability is still required. Painlessly publish your specifications into a software repository to be able to reuse it (to generate client or server code). Etc.
```
Comment (gwenneg) [inline README.md:27]: I don't really understand the meaning of the two last sentences here.

File: README.md [hunk 2]
```diff
@@ -63,18 +63,18 @@ From `openapi-dev-tool` command-line interface, you can:
 
 - **Serve one or several OpenAPI files** specified in [configuration file](#configuration-file) and written in YAML or JSON
 - **Live reloading** to view dynamically your changes in your browser in real-time
-- **Bundle OpenAPI files** to merge the whole of files before serving. By using [OpenAPI remote reference](https://swagger.io/docs/specification/using-ref/), you can work on several files. That facilitates the maintainability and avoid having a large OpenAPI file!
+- **Bundle OpenAPI files** to merge all the files before serving. By using [OpenAPI remote reference](https://swagger.io/docs/specification/using-ref/), you can work on several files. It improves the maintainability and avoids having a large OpenAPI file!
 - **Several documentation viewers embedded** with the possibility to [override the templates](#templates-override) used:
   - Redoc (http://localhost:3000)
   - RedocPro (http://localhost:3000) --> RedocPro to have "try it" feature
```
Comment (gwenneg) [inline README.md:69]: What is the meaning of `RedocPro to have "try it" feature`?

File: README.md (hunk 1 above)
Comment (gigaga) [reply README.md:27]: I like your improvements :)

File: README.md (hunk 2 above)
Comment (gigaga) [reply README.md:69]: RedocPro comes with a "try it" feature which doen't exist in Redoc

**Pull request merged**